Fixed the compile error caused by typo, upstream also need this.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Pending

Index: matchbox-panel-2/applets/startup/startup.c
===================================================================
--- matchbox-panel-2/applets/startup/startup.c	(revision 2109)
+++ matchbox-panel-2/applets/startup/startup.c	(working copy)
@@ -229,7 +229,7 @@
         applet = g_slice_new0 (StartupApplet);
 
         applet->launch_list = NULL;
-        applet->hourglass_show = FALSE;
+        applet->hourglass_shown = FALSE;
 
         /* Create image */
         applet->image = MB_PANEL_SCALING_IMAGE
